server/text_generation_server/models/__init__.py

```diff
@@ -446,8 +467,9 @@ def get_model(
 
     if model_id.startswith("facebook/galactica"):
         return GalacticaSharded(
-            model_id,
-            revision,
+            model_id=model_id,
+            model_class=OPTForCausalLM,
```

Comment:
It seems a bit weird to use a bit weird to use the `GalacticaSharded `constructor and still have to specify that the `model_class=OPTForCausalLM`.

Could this be hidden?

Reply:
I can actually use `CausalLM(.... batch=GalacticaBatch)` instead ! would that work ?

Reply:
Ok!